src/main/markdown/doc/latest/tutorial/i18n.md

```diff
@@ -124,19 +124,20 @@ When you internationalize your application's interface, keep in mind that the la
     *  Open StockWatcher.html.
     *  If the encoding is not already set to UTF-8, copy this code into the `<head>` element.
 
-```
-<meta http-equiv="content-type" content="text/html; charset=UTF-8">
-```
+    ```
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
```

Comment:
Maybe this could be simplified to just `<meta charset="utf-8">` (see https://html.spec.whatwg.org/multipage/semantics.html#character-encoding-declaration)